Readme.md

```diff
@@ -1,628 +1,190 @@
+    cd ./slatedocs
+    docker run --rm --name slate -v $(pwd)/build:/srv/slate/build -v $(pwd)/source:/srv/slate/source slatedocs/slate
+  ```
+  Build result will be generated to `slatedocs/build` folder.
+- Now open this file `slatedocs/build/index.html` - you should see your documentation 
+
+For other ways to build documentation - see slate [getting started][slatedocs getting started] docs
+
+### Start dev server
+The simplest way to build documentation is to use `docker`:
+- install [docker][docker]
+- execute
+  ```shell
+    cd ./slatedocs
+    docker run --rm --name slate -p 4567:4567 -v $(pwd)/source:/srv/slate/source slatedocs/slate serve
+  ```
+- Now open [http://127.0.0.1:4567](http://127.0.0.1:4567) - you should see documentation in your browser
+- Every time you will do changes in the `slatedocs/sources` folder - changes will be available in the browser after 
+  page refresh
+
+For other ways to build documentation - see slate [getting started][slatedocs getting started] docs
+
+### Update slatedocs template to the latest version
+Template is slightly changed but generally it should be easy to update slate template to the latest version
+if necessary. To update template:
+- clone original [slatedocs repo][slatedocs]
+- copy all files and folders from the cloned repo except `.github` and `source/index.html.md` to `slatedocs` 
+  folder and replace files if they exists
+- copy `.github` folder from cloned the repo to the root `.github` folder and replace files if they exists.
+- use some git diff tool to inspect difference and merge changes.
+
+## Github pages
+[Github pages][github pages] used to made documentation site available online.
+To update documentation, please fork this repo and set up github pages to be server from a `gh-pages` branch:
+- In the forked project got to the settings tab and scroll down to the `Github Pages` section.
+- Then select `gh-pages` branch as a source and click `save`
+- After this your site should be available at this url: `https://<username>.github.io/js-visualize/`
+
+## Github actions
+[Github actions][github actions] are used to automatically build and deploy the public site.
+Two actions: `Build` and `Deploy` are created:
+- `Build` action runs on push/pull request to any branch and makes sure that documentation
+  build process executed successfully
+- `Deploy` action runs on push to the `master` branch.
+   it executes documentation build process and then 
+   it executes push of the `slatedocs/buld` folder to the `gh-pages` branch which
+   updates the public documentation if [Github pages](#github-pages) has been set up.
+
+## How to update documentation
+- Fork this repo
+- Set up [Github pages](#github-pages) for your fork  
+- If you already forked the repo earlier - [reset](#how-to-reset-your-fork-to-mirror-original-repo) 
+  your fork's `master` branch to mirror the original repo's `master` branch  
+- Made necessary changes to the `slatedocs/source/index.html.md` file in your fork
```

Comment:
Change "Made necessary " to "Make necessary "

Reply:
fixed